src/main/java/frc/utils/alerts/AlertManager.java

```diff
@@ -0,0 +1,19 @@
+package frc.utils.alerts;
+
+import java.util.ArrayList;
+
+public class AlertManager {
+
+	private static ArrayList<PeriodicAlert> alertList = new ArrayList<>();
+
+	protected static void addAlert(PeriodicAlert periodicAlert) {
+		alertList.add(periodicAlert);
+	}
+
+	public static void logAlerts() {
+		for (PeriodicAlert alert : alertList) {
+			alert.logByCondition();
```

Comment:
the time will update all of the time, make it update once so it will be accurate